test/command/test_cat.rb

```diff
@@ -18,7 +18,7 @@ def setup
     @primary = create_primary
     metadata = @primary.buffer.new_metadata
     @chunk = create_chunk(@primary, metadata, @es)
-    @port = unused_port(protocol: :tcp)
+    @port = unused_port(protocol: :udp)
```

Comment:
```suggestion
    @port = unused_port(protocol: :all)
```

`in_forward` uses both TCP and UDP.

* TCP: for main use
* UDP: for heartbeat

So, maybe we need take care of both here.